<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="de_DE">
<context>
    <name>SplashScreenClass</name>
    <message>
        <location filename="../SplashScreen.ui" line="69"/>
        <source>#terminal_not_available</source>
        <translation>Terminal ist vorübergehend nicht verfügbar</translation>
    </message>
    <message>
        <location filename="../SplashScreen.ui" line="134"/>
        <source>#updater_state_message</source>
        <translation>Wartungsarbeiten</translation>
    </message>
</context>
<context>
    <name>UpdaterApp</name>
    <message>
        <location filename="../UpdaterApp.cpp" line="78"/>
        <source>#error_run_from_temp_dir</source>
        <translation>Fehler beim Start von updater aus dem temporären Ordner</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="79"/>
        <source>#error_connection_to_guard</source>
        <translation>Fehler der Verbindung mit guard.exe</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="80"/>
        <source>#error_unknown_command</source>
        <translation>Unbekannter Befehl</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="81"/>
        <source>#error_second_instance</source>
        <translation>Starten der zweiten Kopie von updater</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="82"/>
        <source>#error_unknown</source>
        <translation>unbekannter Fehler</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="83"/>
        <source>#error_network</source>
        <translation>Netzwerkfehler</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="84"/>
        <source>#error_parse_response</source>
        <translation>Fehler bei der Bearbeitung der Antwort vom Aktualisierungsserver</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="85"/>
        <source>#error_deploy</source>
        <translation>Fehler beim Kopieren der Dateien</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="86"/>
        <source>#error_aborted</source>
        <translation>Durch einen externen Vorgang unterbrochen</translation>
    </message>
    <message>
        <location filename="../UpdaterApp.cpp" line="87"/>
        <source>#error_update_blocked</source>
        <translation>Aktualisierung gesperrt</translation>
    </message>
</context>
</TS>
